packages/peregrine/lib/talons/WishlistPage/useWishlistItem.js

```diff
@@ -94,6 +94,7 @@ export const useWishlistItem = props => {
             // Close the dialogs on success.
             setCurrentDialog(dialogs.NONE);
         } catch (e) {
+            console.error(e);
```

Comment:
Should you add a `NODE_ENV` check around this?

Reply:
Negative. We strip consoles in production.